pyanaconda/modules/security/security.py

```diff
@@ -52,6 +54,8 @@ def __init__(self):
         self.realm_changed = Signal()
         self._realm = RealmData()
 
+        self._realm_required_packages = []
```

Comment:
I would keep packages in the realm data as well. Please, create a new `packages` property.

Reply:
Good point, added. :)